src/main/resources/templates/login.html

```diff
@@ -0,0 +1,18 @@
+<page th:replace="fragments/layout :: page(~{::title},~{::content})">
+    <title>Login</title>
+    <content>
+        <h1>Log in</h1>
+        <br/>
+        <div th:if="${param.error}">
+            Invalid username and password.
+        </div>
+        <div th:if="${param.logout}">
+            You have been logged out.
+        </div>
+        <form th:action="@{/login}" method="post">
+            <div><label> User Name : <input type="text" name="username"/> </label></div>
```

Comment:
User name is an email address - so we should label this "Email address" (you'll have to leave the name attribute as is)

Reply:
@elxsy says: we will leave as username as not to give any clues.